src/azure-cli/azure/cli/command_modules/acs/_completers.py

```diff
@@ -31,12 +31,12 @@ def get_k8s_versions_completion_list(cmd, prefix, namespace, **kwargs):  # pylin
 
 def get_k8s_versions(cli_ctx, location):
     """Return a list of Kubernetes versions available for a new cluster."""
-    from azure.cli.command_modules.acs._client_factory import cf_container_services
+    from azure.cli.command_modules.acs._client_factory import cf_managed_clusters
     from jmespath import search
 
-    results = cf_container_services(cli_ctx).list_orchestrators(location, resource_type='managedClusters').as_dict()
+    results = cf_managed_clusters(cli_ctx).list_kubernetes_versions(location).as_dict()
     # Flatten all the "orchestrator_version" fields into one array
-    return search('orchestrators[*].orchestrator_version', results)
+    return search("values[*].patchVersions.keys(@)[]", results)
```

Comment:
May I ask if this modification will change the output logic and cause a breaking change to customers?

Reply:
This is not a breaking change. Customers will not experience any interruptions.